src/rust/Cargo.toml

```diff
@@ -25,6 +25,7 @@ log = "0.4.20"
 fallible-streaming-iterator = "0.1.9"
 tempfile = "3.8.1"
 reqwest = { version = "0.11.22", optional = true }
+async-trait = "0.1.81"
```

Comment:
why is this added/required?

Reply:
Whoops, missed your response last week.

I need `async-trait` for the new test client: https://github.com/flatgeobuf/flatgeobuf/pull/376/files#diff-b1800ddf53707433e40d9040a9eee56ce39144eaeafca4ca60a5a30893fdcbdbR47

Note that it's ultimately not a *new* dependency, it's just explicit now. It's required by http-range-client (which is why it's required for the test client):

```
$ cargo tree
...
├── http-range-client v0.7.2        
│   ├── async-trait v0.1.81 (proc-macro)
```

Since it's only used in the tests, I've moved it to the "dev" dependencies, so thank you for asking about it.  😉